src/components/BounceRulesContainer/DeleteRuleModal/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import Button from "@sendgrid/ui-components/button";
+import CenterModal from "@sendgrid/ui-components/center-modal";
+import { Column } from "../../Column";
+import { Row } from "../../Row";
+import "../index.scss";
+import { WriteSelectors } from "../selectors";
+
+const DeleteAlertBody = () => (
+  <div>
+    <Row>
+      <Column>
+        <div>
+          <h2>There was an error deleting this rule.</h2>
+          <p>
+            {
```

Comment:
Did we need to put this in an expression? If there were issues with spaces not working you can either do &nbsp; or {' '}